CHANGELOG.md

```diff
@@ -2,14 +2,27 @@
 
 Full documentation for rocBLAS is available at [rocblas.readthedocs.io](https://rocblas.readthedocs.io/en/latest/).
 
-## (Unreleased) rocBLAS 4.0.0
+## rocBLAS 4.0.0 for ROCm 6.0
+### Added
+- Addition of beta API rocblas_gemm_batched_ex3 and rocblas_gemm_strided_batched_ex3
+- Added input/output type f16_r/bf16_r and execution type f32_r support for Level 2 gemv_batched and gemv_strided_batched
+- Added rocblas_status_excluded_from_build to be used when calling functions which require Tensile when using rocBLAS built without Tensile
+### Optimized
+- Trsm performance for small sizes m < 32 && n < 32
 ### Deprecated
-- Atomic operations will be disabled by default in a future release of rocBLAS.
+- Atomic operations will be disabled by default in a future release of rocBLAS
```

Comment:
This wording could be clearer, suggest something like ???
In a future release all internal use of atomic operations will be disabled by default for determinism, but may be enabled with rocblas_set_atomics_mode for performance.